env-template

```diff
@@ -0,0 +1,15 @@
+export csh_auth_state=
+export csh_auth_client_id=
+export csh_auth_client_secret=
+export S3_ACCESS_ID=
+export S3_HOST=
+export S3_SECRET_KEY=
+export csh_auth_jwt_secret=
+export csh_auth_server_host=
+export csh_auth_redirect_uri=
+export DB_URI=
+export LDAP_BIND_PW=
+export LDAP_BIND_DN=
+export LDAP_HOST=
+export GIN_MODE=release
+export BASE_PATH=
```

Comment:
Some of these few like they need some contextualisation

Reply:
I can document these